scripts/references/create_allowed_species.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to create the reference 'allowed_species.json' automatically from the reference 'mlss_conf.xml'."""
+
+
+import json
+import argparse
+import xml.etree.ElementTree
+from typing import List
+
+
+def get_species_list(mlss_conf_file: str) -> List:
+    """Returns a list with all species present in the mlss_conf.xml. Each species will appear only once
+
+    Args:
+        mlss_conf_file: Path to the mlss_conf.xml
+
+    Returns:
+        List of species
+
```

Comment:
Yes I was not sure why in your change you did not write this. Will this give an issue if we generate doc? if not I prefer to keep it because it is more readable that way